sources/common/utils_test.go

```diff
@@ -127,14 +128,14 @@ func TestGetColsAndSchemas(t *testing.T) {
 func TestWorkerPool(t *testing.T) {
 	input := []int{1, 2, 3, 4, 5, 6, 7, 8, 9, 10}
 
-	f := func(i int, mutex *sync.Mutex) TaskResult[int] {
+	f := func(i int, mutex *sync.Mutex) task.TaskResult[int] {
 		sleepTime := time.Duration(rand.Intn(1000 * 1000))
 		time.Sleep(sleepTime)
-		res := TaskResult[int]{Result: i, Err: nil}
+		res := task.TaskResult[int]{Result: i, Err: nil}
 		return res
 	}
 
-	r := RunParallelTasksImpl[int, int]{}
+	r := task.RunParallelTasksImpl[int, int]{}
```

Comment:
Looks to me that this entire test should now be in `task_test.go` in the `task` package?

Reply:
Yes, I'll move them. Missed that. thamks